features/cellular/framework/targets/QUECTEL/BG96/QUECTEL_BG96_CellularStack.cpp

```diff
@@ -299,3 +334,57 @@ nsapi_size_or_error_t QUECTEL_BG96_CellularStack::socket_recvfrom_impl(CellularS
 
     return recv_len;
 }
+
+nsapi_error_t QUECTEL_BG96_CellularStack::gethostbyname(const char *host, SocketAddress *address,
+                                                        nsapi_version_t version, const char *interface_name)
```

Comment:
`interface_name` is an unused variable, so please `void` it to prevent compiler warning.